packages/perseus/src/widgets/__tests__/radio.test.ts

```diff
@@ -828,13 +828,15 @@ describe("multi-choice question", () => {
 
     it.each(invalid)(
         "should reject an invalid answer - test #%#",
-        (...choices) => {
+        async (...choices) => {
             // Arrange
             const {renderer} = renderQuestion(question, apiOptions);
 
             // Act
             const option = screen.getAllByRole("checkbox");
-            choices.forEach(async (i) => await userEvent.click(option[i]));
+            for (const i of choices) {
+                await userEvent.click(option[i])
+            }
```

Comment:
More and more I'm becoming a fan of using this type of for loop over `.forEach()`. To me, `.forEach()` isn't a higher-order function like other, more common ones (`.map()`, `.filter()`, etc).

Reply:
Yeah, the `forEach` callback needs to have a side effect in order to do anything useful. So there's not much advantage over a for loop. The one case where I might use it is at the end of a functional pipeline, like:

```js
things.map(...)
  .filter(...)
  .map(...)
  .forEach(console.log)
```